### PR TITLE
Feature: description

### PR DESCRIPTION
**Pull request by vadymmarkov**
@hyperoslo/everyone This is an initial project overview, describing main goals, project structure and the ways we can collaborate together. Feel free to add your corrections, comments and suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,126 @@
-# hyper-radar
-Research and development at Hyper
+# Research and development at Hyper
+
+We at [Hyper](https://www.hyper.no) are actively doing research in different
+areas of information technology. It's in our DNA to be curious about new trends
+and developments, as it's the only way to survive in our fast-changing world.
```
Comment (onmyway133) [inline README.md]: How about "in this fast-changing world"? Here "our" seems to refer to Hyper

**Review by zenangst — APPROVED** [review]
@vadymmarkov 
687474703a2f2f6d65646961302e67697068792e636f6d2f6d656469612f67655977746f644239416949302f67697068792e676966

File: README.md [hunk 2]
```diff
@@ -1,2 +1,126 @@
...
+## 🎤 Stage
+
+Have you recently been a speaker at a local meetup or at a conference?
+Open the [Stage](https://github.com/hyperoslo/hyper-radar/blob/master/Stage)
+folder and share your keynote, so everyone can enjoy the slides and re-visit it
+on any day, at any time.
+
+## 🦄 Inspiration
+
+Use (https://github.com/hyperoslo/hyper-radar/blob/master/inspiration.md) to
```
Comment (onmyway133) [inline README.md:92]: This will be the biggest file 😉

File: README.md (hunk 1 above)
Comment (zenangst) [reply README.md]: `it's the only way to survive in an fast-changing industry.`

File: README.md (hunk 2 above)
Comment (zenangst) [reply README.md:92]: Love the unicorn! 👍

Comment (vadymmarkov) [reply README.md:92]: I don't know yet how useful it is to have a file with a list of links, but we'll see how it goes, maybe  we don't need it at all, or maybe it makes sense to slip it up to multiple files.

Comment (zenangst) [reply README.md:92]: Let's just see where this takes us 😎

**Comment by onmyway133** [comment]
🚀

**Pull request merged**